src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,81 @@
+package homework_2.random_chars_table;
+
+import homework_2.ConsoleSafeReader;
+
+import java.util.InputMismatchException;
+import java.util.Random;
+import java.util.Scanner;
+
+public class RandomCharsTable {
```

Comment:
Most of my tests fail when run together, but pass if run independently. Most probably, it is because you use a lot of static. Make objects where possible.